authors/settings.py

```diff
@@ -136,8 +136,42 @@
 REST_FRAMEWORK = {
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
-
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
+        'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
```

Comment:
Are this changes related to the migration from 1.10 to 2.1 ??

Reply:
These were also some of the changes that were applied to test whether the application is working (as it was originally). It was a debugging attempt that was meant to make the app run. It was not meant to be pushed, we shall be vigilant from here onwards.